### PR TITLE
Add Package-Analyzer Job Template and some changes

### PR DESCRIPTION
**Pull request by bissenbay**
Added Package-Analyzer Job Template

### CODE REVIEW

File: openshift/packageAnalyzerSyncJob-template.yaml [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    displayName: ImageStream Registry
+    required: true
+    name: IMAGE_STREAM_REGISTRY
+    value: "docker-registry.default.svc:5000"
+
+  - description: Namespace the ImageStream to be use lives in
+    displayName: ImageStream Project
+    required: true
+    name: IMAGE_STREAM_NAMESPACE
+    value: "thoth-infra-stage"
```
Comment (bissenbay) [inline openshift/packageAnalyzerSyncJob-template.yaml]: Package-Analyzer is deployed on **thoth-test-core**, but I can deploy to **thoth-infra-stage** if there is a need.

File: openshift/packageAnalyzerSyncJob-template.yaml [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+            graph-sync-document-id: "${THOTH_SYNC_DOCUMENT_ID}"
+            graph-sync-type: package-analyzer
+            mark: cleanup
+            ttl: "1h"
+        spec:
+          containers:
+            - name: graph-sync
+              image: "${IMAGE_STREAM_REGISTRY}/${IMAGE_STREAM_NAMESPACE}/graph-sync-job:${IMAGE_STREAM_TAG}"
+              env:
+                - name: THOTH_ONLY_PACKAGE_ANALYSIS_DOCUMENTS
```
Comment (fridex) [inline openshift/packageAnalyzerSyncJob-template.yaml]: Can be the variable renamed to `THOTH_ONLY_PACKAGE_ANALYZER_DOCUMENTS` to be consistent?

File: openshift/packageAnalyzerSyncJob-template.yaml [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+                - name: THOTH_SYNC_DOCUMENT_ID
+                  value: "${THOTH_SYNC_DOCUMENT_ID}"
+                - name: THOTH_FORCE_SYNC
+                  value: "1"
+                - name: THOTH_DEPLOYMENT_NAME
+                  valueFrom:
+                    configMapKeyRef:
+                      key: storage-bucket-name
+                      name: thoth
+                - name: THOTH_SYNC_OBSERVATIONS
```
Comment (fridex) [inline openshift/packageAnalyzerSyncJob-template.yaml]: I think this one will not be used (I've just realized it when looking at the PR, its not your issue as it was already introduced).

File: openshift/packageAnalyzerSyncJob-template.yaml (hunk 1 above)
Comment (fridex) [reply openshift/packageAnalyzerSyncJob-template.yaml]: Keep it just in thoth-test-core for now.

Comment (bissenbay) [reply openshift/packageAnalyzerSyncJob-template.yaml]: @fridex I changed it to **thoth-test-core**

Comment (fridex) [reply openshift/packageAnalyzerSyncJob-template.yaml]: I meant - keep it deployed only in thoth-test-core. The configuration option you changed is adjusted from our deployment scripts.

File: openshift/packageAnalyzerSyncJob-template.yaml (hunk 2 above)
Comment (bissenbay) [reply openshift/packageAnalyzerSyncJob-template.yaml]: For sure, that is better, changed.

File: openshift/packageAnalyzerSyncJob-template.yaml (hunk 3 above)
Comment (bissenbay) [reply openshift/packageAnalyzerSyncJob-template.yaml]: I deleted from all templates, is that OK? Or should I have deleted it only from **package-analyzer job template**?

Comment (fridex) [reply openshift/packageAnalyzerSyncJob-template.yaml]: It's ok. Next time please open a separate PR :+1:

Comment (bissenbay) [reply openshift/packageAnalyzerSyncJob-template.yaml]: OK, thanks, got it!

File: openshift/packageAnalyzerSyncJob-template.yaml (hunk 1 above)
Comment (bissenbay) [reply openshift/packageAnalyzerSyncJob-template.yaml]: Got it, let me bring **thoth-infra-stage** back.

**Comment by bissenbay** [comment]
@fridex There was a conflict which I resolved. Could you please take a look? Thanks!

**Pull request merged**